confgenerator/confmerger.go

```diff
@@ -20,7 +20,7 @@ func MergeConfFiles(userConfPath, platform string, builtInConfStructs map[string
 	builtInStruct := builtInConfStructs[platform]
 
 	// Read the built-in config file.
```

Comment:
Might want to adjust this comment, since it's no longer accurate. Maybe:

```suggestion
	// Start with the built-in config.
```

?